frontend/src/components/Delete.js

```diff
@@ -0,0 +1,25 @@
+import React, { useEffect } from "react";
+import { Button } from "react-bootstrap";
+import { useParams } from "react-router-dom";
+import axios from "axios";
+
+const Delete = () => {
+  const { id } = useParams();
+  const url = `http:localhost:8000/user/${id}`;
+  const deleteHandler = async () => {
+    try {
+      await axios.delete(url);
+      alert("deleted successfully");
```

Comment:
lets implement reusable model component pop up instead of alert.
for both error and success